src/Features/LanguageServer/Microsoft.CommonLanguageServerProtocol.Framework/HandlerProvider.cs

```diff
@@ -130,7 +134,13 @@ static string GetRequestHandlerMethod(Type handlerType, Type? requestType, Type
                 }
             }
 
-            return methodAttribute.Method;
+            var languages = methodAttribute.Languages;
+            if (languages.Length == 0)
+            {
+                languages = new string[] { string.Empty };
```

Comment:
this is... strange.  it supports one language, but that language is empty?  i don't get/like it.